acvm/src/compiler.rs

```diff
@@ -106,3 +106,61 @@ pub fn compile(
         return_values: acir.return_values,
     })
 }
+
+#[test]
+fn compile_api_compiles_with_backend() {
+    use crate::ProofSystemCompiler;
+
+    compile(
+        Circuit::default(),
+        Language::R1CS,
+        |opcode| DummyProofSystem.opcode_supported(opcode),
```

Comment:
So is this never called because it is R1CS? I was confused when `opcode_supported` has a panic but it is used here.

Reply:
This is never called as the `Circuit` doesn't contain any opcodes to check.